LZW_Alg/LZW/LZW.cs

```diff
@@ -0,0 +1,110 @@
+using TransformerForEncode;
+using TransformerForDecode;
+namespace LZW;
+public class ALZW
+{
+    const int Bytes = 256;
+    private int num1 = 256;
+    private int num2 = 256;
+
+    public byte[] Encode(byte[] bitsOfOriginalFile)
+    {
+        var dictionary = new Trie();
+        var number = Bytes;
+
+        for (int i = 0; i<Bytes; ++i)
+        {
+            List<byte> a=  new List<byte>{(byte)i};
```

Comment:
```suggestion
            List<byte> a = new List<byte>{(byte)i};
```
советую поставить какой-нибудь линтер, чтобы он форматировал код за тебя